airbyte-integrations/connectors/destination-exasol/src/main/java/io/airbyte/integrations/destination/exasol/ExasolSqlOperations.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.exasol;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.jdbc.JdbcSqlOperations;
+import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
+
+public class ExasolSqlOperations extends JdbcSqlOperations {
+
+  public static final String COLUMN_NAME_AB_ID =
+      "\"" + JavaBaseConstants.COLUMN_NAME_AB_ID.toUpperCase() + "\"";
+  public static final String COLUMN_NAME_DATA =
+      "\"" + JavaBaseConstants.COLUMN_NAME_DATA.toUpperCase() + "\"";
+  public static final String COLUMN_NAME_EMITTED_AT =
+      "\"" + JavaBaseConstants.COLUMN_NAME_EMITTED_AT.toUpperCase() + "\"";
+
+  @Override
+  public String createTableQuery(final JdbcDatabase database, final String schemaName, final String tableName) {
+    String query = String.format(
+        "CREATE TABLE IF NOT EXISTS %s.%s ( \n"
+            + "%s VARCHAR(64),\n"
+            + "%s VARCHAR(2000000),\n"
```

Comment:
the varchar length of '2000000' seems quite limiting when it comes to storing data. Isn't there some other type for storing a variable length of data or a json data type?

Reply:
`VARCHAR(2000000)` is the largest data type supported by Exasol. There is no special json type.
See https://docs.exasol.com/db/latest/sql_references/data_types/datatypedetails.htm#StringDataTypes

Reply:
Okay then pls document this in the connector documentation if not already

Reply:
Done.